olad/PluginManager.cpp

```diff
@@ -88,10 +88,15 @@ void PluginManager::LoadAll() {
     set<ola_plugin_id>::const_iterator set_iter = conflict_list.begin();
     for (; set_iter != conflict_list.end(); ++set_iter) {
       if (STLContains(m_enabled_plugins, *set_iter)) {
+        AbstractPlugin *conflicting_plugin = GetPlugin(*set_iter);
         OLA_WARN << "Skipping " << plugin->Name()
-                 << " because it conflicts with "
-                 << GetPlugin(*set_iter)->Name()
-                 << " which is also enabled";
+                 << " because it conflicts with ";
+        if (conflicting_plugin) {
+          OLA_WARN << conflicting_plugin->Name();
+        } else {
+          OLA_WARN << "plugin id " << *set_iter;
```

Comment:
This is going to split the message across 3 lines.

I'd just do:

<< " because it conflicts with "
<< PluginIdentifier(*set_iter) << "which is also enabled";

then

string PluginIdentifier(plugin_id id) {
  AbstractPlugin *plugin = GetPlugin(*set_iter);
  return plugin ? plugin->Name() : "plugin_id: " + id;
}